plan/plan_loader_test.go

```diff
@@ -65,3 +66,65 @@ func TestLoadOneFile(t *testing.T) {
 	}
 	assert.Equal(t, gotPlans, expectPlans)
 }
+
+func TestPlan_ShouldReturnDeepCopyOfPlan(t *testing.T) {
```

Comment:
Please don't use underscores in test names like this; it's not Blip convention (and, I'd argue, not common in Go--at least not the Go I've seen). It can be shorter (Go loves short), like `TestDeepCopy` (the "Plan" part is implied given this pkg is narrowly scoped around plans, so all tests relate to plans).

Reply:
Gotcha. I agree that if it's not a convention in this code base, then I'll avoid it. But FWIW, it is quite common and part of examples listed in [the examples of the official godoc testing overview](https://pkg.go.dev/testing#hdr-Examples).